### PR TITLE
Chore: get rid of russian text in tests

### PR DESCRIPTION
**Pull request by iBat**

### CODE REVIEW

File: testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/toolbarModule.tests.js [hunk 1]
```diff
@@ -1988,12 +1988,10 @@ testModule('tables', simpleModuleConfig, function() {
 testModule('Toolbar localization', simpleModuleConfig, function() {
     const messages = {
         'ru': {
```
Comment (nightskylark) [inline testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/toolbarModule.tests.js]: Don't we need to change a locale key too?

File: testing/tests/DevExpress.ui.widgets/pager.tests.js [hunk 2]
```diff
@@ -1632,14 +1632,12 @@ function() {
                 'dxPager-prevPage': 'test Previous Page',
                 'dxPager-nextPage': 'test Next Page',
             },
-            'ru': {
```
Comment (nightskylark) [inline testing/tests/DevExpress.ui.widgets/pager.tests.js:1635]: Same here

Comment (iBat) [reply testing/tests/DevExpress.ui.widgets/pager.tests.js:1635]: It is line before changes. It has been changed to 'fr'

Comment (nightskylark) [reply testing/tests/DevExpress.ui.widgets/pager.tests.js:1635]: My bad :)

**Review by pomahtri — APPROVED** [review]

**Review by AlisherAmonulloev — APPROVED** [review]

**Pull request merged**